src/nginx/t/ApiManager.pm

```diff
@@ -204,6 +204,10 @@ sub get_bookstore_service_config {
   return read_test_file("testdata/bookstore.pb.txt");
 }
 
+sub get_bookstore_service_config_2 {
```

Comment:
could you add comment to state the different between bookstore config 1 and 2.  why 2 is needed?